pkg/console/install_panels.go

```diff
@@ -40,6 +40,7 @@ var (
 	once          sync.Once
 	userInputData = UserInputData{
 		NTPServers: "0.suse.pool.ntp.org",
+		DNSServers: "1.1.1.1",
```

Comment:
1.1.1.1 is fine to me. Raise it up here for others to double check this is what we want by default.

Reply:
I'd prefer to have no default DNS servers in this field. Users should manually enter servers if they need it.
We don't host DNS servers like `1.1.1.1` or `8.8.8.8`, hard-coding them in the system is misuse.

Reply:
Okay. I remove it. Thank you.